app/views/users/registrations/created.html.erb

```diff
@@ -10,8 +10,7 @@
     <%= ts("If you don't hear from us within 24 hours, and you don't find our email in your spam filter, please <a href='%{support_path}'>drop us a line at Support</a>.", support_path: new_feedback_report_path).html_safe %>
   </p>
   <p>
-    <% # We do days_to_purge_unactivated * 7 because it's actually weeks %>
-    <%= ts("<strong>Important!</strong> You must verify your account within %{days_before_purge} days, or your registration will expire.", days_before_purge: AdminSetting.first.days_to_purge_unactivated? ? (AdminSetting.first.days_to_purge_unactivated * 7) : ArchiveConfig.DAYS_TO_PURGE_UNACTIVATED).html_safe %>
+    <%= ts("<strong>Important!</strong> You must verify your account within %{days_before_purge}, or your registration will expire.", days_before_purge: distance_of_time_in_words(Devise.confirm_within)).html_safe %>
```

Comment:
Yeah, the days/weeks mismatch is definitely confusing. I'm going to check with the development team to see if anyone has a strong preference for weeks or days, and I'll get back to you tomorrow.